apps/condo/domains/miniapp/schema/B2BAccessToken.js

```diff
@@ -0,0 +1,280 @@
+/**
+ * Generated by `createschema miniapp.B2BAccessToken 'sessionId:Text; context:Relationship:B2BAppContext:CASCADE; rightSet:Relationship:B2BAppAccessRightSet:CASCADE; expiresAt:Text;'`
+ */
+
+const crypto = require('crypto')
+
+const cookieSignature = require('cookie-signature')
+const dayjs = require('dayjs')
+const get = require('lodash/get')
+const has = require('lodash/has')
+const isEmpty = require('lodash/isEmpty')
+const isNil = require('lodash/isNil')
```

Comment:
Are u really need utils like that?)

(Not the biggest fan of 50 imported mini-utils for simple tasks)